src/clj/villagebook/field/db.clj

```diff
@@ -0,0 +1,9 @@
+(ns villagebook.field.db
+  (:require [clojure.java.jdbc :as jdbc]
+            [villagebook.config :as config]))
+
+(defn create-fields!
+  ([category-id fields]
+   (create-fields! (config/db-spec) category-id fields))
+  ([conn category-id fields]
+   (jdbc/insert-multi! conn :fields (map #(assoc % :category_id category-id) fields))))
```

Comment:
You can potentially move this data massaging `(map #(assoc % :category_id category-id) fields))`
to the field.models namespace. 
The db namespace should just do the db operations.